app/images/remove.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
There is an icons.sketch maybe use it, add your icon and export a png? Wonder who did this .sketch, I do not have sketch :)